components/src/dynamo/rl_admin/service.py

```
@@ -0,0 +1,434 @@
+        # Extract completion token IDs from logprobs
+        for choice in response_data.get("choices", []):
+            lp = choice.get("logprobs")
+            if lp and lp.get("content"):
+                logprobs_content = lp["content"]
+                completion_ids = _extract_completion_ids_from_logprobs(logprobs_content)
+                assert len(completion_ids) == len(logprobs_content), (
+                    f"Token alignment: {len(completion_ids)} ids vs "
+                    f"{len(logprobs_content)} logprobs"
+                )
+                choice["token_ids"] = completion_ids
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)